.woodpecker.yml

```diff
@@ -6,7 +6,9 @@ pipeline:
   check_ansible_format:
     image: alpine:3
     commands:
-      - apk add ansible
+      - apk add ansible ansible-lint
       - ansible-playbook lemmy.yml --syntax-check
       - ansible-playbook lemmy-almalinux.yml --syntax-check
       - ansible-playbook uninstall.yml --syntax-check
+      - ansible-lint --version
+      - ansible-lint --strict lemmy.yml lemmy-almalinux.yml uninstall.yml
```

Comment:
I'd consider not using `--strict` to avoid CI conflicts with `prettier` (which is *very* opinionated). 

Is there any reason for the `ansible-lint --version` here, or is it just a leftover debug line?

Reply:
`ansible-lint --version` was added for my CI troubleshooting, removed now.

do you know of any conflicts with `prettier`?

I've included strict to ensure warnings like [`jinja[spacing]`](https://ansible.readthedocs.io/projects/lint/rules/jinja/) would be addressed/flagged:

```
jinja[spacing]: Jinja2 spacing could be improved: {{ 32767 |random(start=1024) }} -> {{ 32767 | random(start=1024) }} (warning)
lemmy-almalinux.yml:110 Jinja2 template rewrite recommendation: `{{ 32767 | random(start=1024) }}`.

jinja[spacing]: Jinja2 spacing could be improved: docker.io/dessalines/lemmy-ui:{{ lemmy_ui_version | default(lemmy_version | default(lookup('file', 'VERSION')))}} -> docker.io/dessalines/lemmy-ui:{{ lemmy_ui_version | default(lemmy_version | default(lookup('file', 'VERSION'))) }} (warning)
lemmy-almalinux.yml:127 Jinja2 template rewrite recommendation: `docker.io/dessalines/lemmy-ui:{{ lemmy_ui_version | default(lemmy_version | default(lookup('file', 'VERSION'))) }}`.

jinja[spacing]: Jinja2 spacing could be improved: docker.io/dessalines/lemmy:{{ lemmy_version | default( lookup('file', 'VERSION') )}} -> docker.io/dessalines/lemmy:{{ lemmy_version | default(lookup('file', 'VERSION')) }} (warning)
lemmy-almalinux.yml:127 Jinja2 template rewrite recommendation: `docker.io/dessalines/lemmy:{{ lemmy_version | default(lookup('file', 'VERSION')) }}`.

[…]

                        Rule Violation Summary
 count tag                      profile    rule associated tags
[…]
    37 jinja[spacing]           basic      formatting (warning)
```

If necessary due to `prettier` conflicts, individual linter checks could still be ignored.

Reply:
I am unaware of any conflicts off-hand. However, it's almost guaranteed to happen.

`prettier` does a good job explaining what it is trying to achieve  [here](https://prettier.io/docs/en/comparison.html) and why we shouldn't mix it with `--strict`. 

> In other words, use Prettier for formatting and linters for catching bugs!

Reply:
`prettier` isn't aware of ansible-specific formatting, such as jinja templated strings.
With the current yaml, `jinja[spacing]` was the only type of warning emitted, and imo it's reasonable for this to fail CI.
As of right now, on the latest version of `ansible-lint`, the following rules only emit warnings:

> Current version default value is: experimental, jinja[spacing], fqcn[deep]

I don't see how this could in any way conflict with `prettier`.

While it probably makes sense to ignore `experimental` checks, and `fqcn[deep]` checks are not relevant to this repo at all (currently), `jinja[spacing]` is relevant.

A better alternative seems to be using `--warn-list experimental`, which I've updated this PR to.

Reply:
I just took a closer look, and you're right. I don't think there will be any conflicts (well, any that would cause a loop with the CI). 

I think `--warn-list experimental` is a good alternative.